backend/projects/models.py

```diff
@@ -1,3 +1,36 @@
 from django.db import models
+from profiles.models import Startup, Investor
 
 # Create your models here.
+class ProjectStatus(models.Model):
+    status = models.CharField(max_length=150)
+
+    def __str__(self):
+        return self.status
+
+class StartupProject(models.Model):
+    startup = models.ForeignKey(Startup, on_delete=models.CASCADE, related_name='projects')
+    subject = models.CharField(max_length=150)
+    idea = models.TextField()
+    description = models.TextField(blank=True)
+    website = models.URLField(blank=True)
+
+    investment_needed = models.IntegerField(default=0)
+    views = models.IntegerField(default=0)
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    status = models.ForeignKey(ProjectStatus, on_delete=models.SET_NULL, related_name='status')
```

Comment:
Problem: related_name='status' is a conflicting and uninformative name. It creates a callback projectstatus.status.all()

update on
related_name='project_statuses'